### ISSUE TITLE
We pay but we don't get support

### ISSUE THREAD

**Issue by arrtchiu** [docker/hub-beta-feedback#1731]
Like the title says.
Treat your customers with a bit of respect and actually reply to them.

**Comment by zdm** [comment]
I am also wrote to support 3 times but get no response!

**Comment by mitar** [comment]
Same here. I think Docker as a company is probably going into the ground. :-(

**Comment by manuasir** [comment]
I'm concerning about this, can you confirm that they still don't provide support to you?

**Comment by arrtchiu** [comment]
Nope, nothing. Don't use Docker for anything except development work. Production should stay far away from it!

**Comment by rafaelthca** [comment]
I don't pay, am waiting for a response for 3 weeks and nothing. Knowing that even customers don't get their tickets responded to probably means that my issue will never be solved.

**Comment by timja** [comment]
I've been waiting for a month 😢 I don't pay but 😞

**Comment by Suika** [comment]
get ready for 3+ months

**Comment by manuasir** [comment]
They recently updated the full website UI, and after linking the GitHub account again everything works properly.

**Comment by senseysensor** [comment]
The same here! how should I have any support? we also pay money for this company.

**Comment by arrtchiu** [comment]
Here we are two years later and they won't even comment on their own support, but they've found the time to go create a bot that closes tickets...
Tells you quite a lot about the future of Docker as a company.

**Comment by mitar** [comment]
Moved to GitLab. Feel free to close it.

**Comment by sheltongraves** [comment]
Hey!

Yes, we offer support across all plan levels. You can check out the details here:[https://www.docker.com/support/#pro](url)
